### PR TITLE
feat: add behaviour for dropdown change

### PR DESCRIPTION
**Pull request by IsaacHook**

### CODE REVIEW

File: src/nodeUpdate.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { Schema, Kinds, Node, GetParams } from "./state.js"
+
+import { CanvasContext } from "./context.js"
+import { EditorAction, updateNode } from "./actions.js"
+
+export type NodeUpdateBehaviour = <S extends Schema>(node: Node<S>, action: string) => void
+
+export function makenodeUpdateBehaviour<S extends Schema>(
```
Comment (alex-lucas) [inline src/nodeUpdate.ts]: Nit: could be camelcase similar to `makeNodeDragBehavior`.

File: src/nodeUpdate.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import type { Schema, Kinds, Node, GetParams } from "./state.js"
+
```
Comment (alex-lucas) [inline src/nodeUpdate.ts]: Nit: Could we remove this line?

File: src/editor/EditableCanvas.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ import { GraphEdge } from "../Edge.js"
 import { makeInputDragBehavior } from "../inputDrag.js"
 import { makeOutputDragBehavior } from "../outputDrag.js"
 import { makeNodeDragBehavior } from "../nodeDrag.js"
+import { makenodeUpdateBehaviour } from "../nodeUpdate.js"
 import { makeParamUpdateBehavior } from "../paramUpdate"
```
Comment (alex-lucas) [inline src/editor/EditableCanvas.tsx]: Nit: Could we remove this line?

**Review by alex-lucas — COMMENTED** [review]
Just a few small things

**Review by alex-lucas — APPROVED** [review]

**Review by reecelondon — APPROVED** [review]

**Pull request merged**